css-contain-3/Overview.bs

```diff
@@ -460,6 +461,67 @@ Container Queries: the ''@container'' rule</h2>
 	so it is not always possible to merge the individual <<container-query-list>>s
 	into a single query.
 
+<h3 id="animated-containers">
+Animated Containers</h3>
+
+	A change in the evaluation of a [=container query=] must be treated as a [=style change event=],
+	even when the change occurred because of [=effect values|animation effects=].
+
+	<div class=example>
+		A transition on a sibling element can indirectly affect the size of a
+		container, triggering [=style change events=] whenever container queries
+		change their evaluation as a result:
+
+		<pre class=lang-css>
+			main {
+			  display: flex;
+			  width: 300px;
+			}
+
+			#container {
+			  container: inline-size;
+			  flex: 1;
+			}
+
+			/* Resolved width is initially 200px, but changes as the transition
+			   on #sibling progresses. */
+			#inner {
+			  transition: 1s background-color;
+			  background-color: tomato;
+			}
+
+			/* When this container query starts (or stops) applying, a transition
+			   must start on background-color on #inner. */
+			@container (width <= 150px) {
+			  #inner {
+			    background-color: skyblue;
+			  }
+			}
+
+			#sibling {
+			  width: 100px;
+			  transition: width 1s;
+			}
+
+			#sibling:hover {
+			  width: 200px;
+			}
+		</pre>
+
+		<pre class=lang-html>
+			&lt;main>
+			  &lt;div id=container>
+			    &lt;div id=inner>Inner&lt;/div>
+			  &lt;/div>
+			  &lt;div id=sibling>Sibling&lt;/div>
+			&lt;/main>
+		</pre>
+
+	</div>
+
+	This also applies for changes in [=computed values=] caused by [=query length=] units.
```

Comment:
"This also applies" is a bit vague (and could become worse if more text gets inserted nearby).

Perhaps, instead:

"Changes in computed values caused by query length units must also be part of a style change event"

?

Reply:
Good point, done.